doc/source/contributing_code.rst

```diff
@@ -60,6 +60,10 @@ both modules in development mode.
 
 You can do this either with :ref:`conda <dev-with-conda>` or :ref:`pip <dev-with-pip>`.
 
+.. note::
+    If you are a first time contributor and/or don't have a lot of experience managing
+    your own Python virtual environments, we suggest to use :code:`conda`.
```

Comment:
```suggestion
    your own Python virtual environments, we **strongly** suggest using :code:`conda`.
```

*Really really strongly* :p

Reply:
Given that we _strongly_ recommend `conda`, should we remove the information about `virtualenvwrapper`? If someone wants to go down the `virtualenv` route, probably they know what they are doing and know about it already?